### DISCUSSION TITLE
Flag non relevant issues

### DISCUSSION THREAD

**Discussion by runnerpaul** [community/community#22868] [New to GitHub]
<p>I’m on one of several support teams that deals with issues on a particular GitHup repository.</p>
<p>Our process is to check all issues raised and deal with any that are applicible to our team. We just ignore any other issues.</p>
<p>Is there any way that an issue could be flagged as having been checked by our team and it was determined that this is not of us? We currently add a tag to any relevant issues. I had considered adding a similar tag to identify non relevant issues but would like a better option if one exists.</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi <a class="mention" href="/runnerpaul">@runnerpaul</a>,</p>
<p>Thanks for being part of the GitHub Community Forum. </p>
<p>There isn’t currently a native way to do what you’re specifically asking. I’d recommend a label or comment acknowledgement system for handling these kinds of situations. As you mentioned, creating a label to identify non-relevant issues would be an option.</p>
<p>Cheers!</p>
